dal/utilities.go

```diff
@@ -0,0 +1,30 @@
+package dal
+
+import (
+	"strconv"
+	"strings"
+)
+
+func GetPriceRange(priceRangeStr string) []float64 {
```

Comment:
You must return the error, and let the caller of the function to choose how to deal with it